tests/test.py

```diff
@@ -72,14 +72,9 @@ def setUp(self):
                           's3': {'static-bucket-name': 'moj-test-dev-static'}}}
         yaml.dump(config, open(self.env.config, 'w'))
 
-        self.aws_config = mock.Mock()
-        self.aws_config.aws_access = 'aws_access_key'
-        self.aws_config.aws_secret = 'aws_secret_key'
-        self.aws_config.region = 'aws_region'
-
         self.stack_name = '{0}-{1}'.format(self.env.application,
                                            self.env.environment)
-        self.cf = cloudformation.Cloudformation(self.aws_config)
+        self.cf = cloudformation.Cloudformation('the-profile', 'aws_region')
```

Comment:
I don't know what `the-profile` refers to here (or, indeed, everywhere else); is that something we can put into a variable, or make clearer?